### PR TITLE
Cleancode

### PR DESCRIPTION
**Pull request by tomik27**
První pokus.

### CODE REVIEW

File: INPTPZ1/ComplexNumber.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        {        
+            public double Re { get; set; }
+            public double Im { get; set; }
+
+          public   ComplexNumber(double Re, double Im) {
+                this.Re = Re;
+                this.Im = Im;
+            }
+
+            public readonly static ComplexNumber Zero = new ComplexNumber(0, 0);
```
Comment (rodi0878) [inline INPTPZ1/ComplexNumber.cs]: constants should be at a top of class

File: INPTPZ1/ComplexNumber.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace INPTPZ1 {
+
+    namespace Mathematics {
+        public class ComplexNumber
+        {        
+            public double Re { get; set; }
+            public double Im { get; set; }
+
+          public   ComplexNumber(double Re, double Im) {
```
Comment (rodi0878) [inline INPTPZ1/ComplexNumber.cs]: use a proper "format code" tool, there is a mess in blank characters

File: INPTPZ1/ComplexNumber.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            public double GetAbsoluteValue()
+            {
+                return Math.Sqrt( Re * Re + Im * Im);
+            }
+
+            public double GetAngleInRadian() {
+                return Math.Atan(Im / Re);
+            }
+
+            public ComplexNumber Add(ComplexNumber addition)
```
Comment (rodi0878) [inline INPTPZ1/ComplexNumber.cs]: please reorder methods so that all mathematical operations are close together

File: INPTPZ1/NewtonFractal.cs [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            for (int i = 0; i < image.width; i++) {
+                for (int j = 0; j < image.height; j++) {
+                    ComplexNumber coordinatesOfPixel = FindCoordinatesOfPixel(j,i);
+                    HandleZeroValueOfCoordinates(ref coordinatesOfPixel);
+                     FindSolution(ref coordinatesOfPixel);
+                    int indexOfRoot = FindRootIndex(coordinatesOfPixel);
+                    image.ColorizePixel(indexOfRoot, j, i);
+                }
+            }
+          //  image.saveImage();
```
Comment (rodi0878) [inline INPTPZ1/NewtonFractal.cs]: delete dead code

File: INPTPZ1/NewtonFractal.cs [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                    indexOfRoot = i;
+                }
+            }
+            if (!known) {
+                roots.Add(root);
+                indexOfRoot = roots.Count;
+            }
+            return indexOfRoot;
+        }
+       public void setPolynomial() {
```
Comment (rodi0878) [inline INPTPZ1/NewtonFractal.cs]: name of methods doesnt follow C# naming conventions; also set method without parameters is strange, find a better name

File: INPTPZ1/Point.cs [hunk 6]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace INPTPZ1 {
+  
+    class Point {
+        public double x { get; set; }
```
Comment (rodi0878) [inline INPTPZ1/Point.cs]: also broken naming conventions for properties

**Review by rodi0878 — CHANGES_REQUESTED** [review]
Please look at mentioned problems. There is quite a few things to clean up... Also use "format code" on all source code files now its in a complete mess.

It would be nice to do some cleaning in tests part of project.

**Comment by tomik27** [comment]
Problems fixed.

**Comment by rodi0878** [comment]
Accepted

**Pull request closed without merging**